lua/wire/zvm/tests/file_example.lua

```diff
@@ -0,0 +1,29 @@
+CPUTest = {}
```

Comment:
Don't really like the idea of exposing a global here, I never liked garry doing this with TOOL and whatnot. Why not just return the CPUTest table at the bottom and have this be a local variable? `include` returns a value after all

Reply:
Didn't know at the time, have corrected it.